File: app/src/main/java/com/vkpapps/thunder/interfaces/OnFileRequestPrepareListener.java
package com.vkpapps.thunder.interfaces;

import com.vkpapps.thunder.model.RawRequestInfo;

import java.util.List;
/***
 * @author VIJAY PATIDAR
 */
public interface OnFileRequestPrepareListener {
    void sendFiles(List<RawRequestInfo> requests, int type);
}

File: app/src/main/java/com/vkpapps/sendkr/interfaces/OnFileRequestPrepareListener.kt
package com.vkpapps.sendkr.interfaces

import com.vkpapps.sendkr.model.RawRequestInfo

/***
 * @author VIJAY PATIDAR
 */
interface OnFileRequestPrepareListener {
    fun sendFiles(requests: List<RawRequestInfo>)
}